docs/_interface/merge.md

```diff
@@ -3,93 +3,91 @@ title: The Merge
-case for Ropsten, Geth will:
+![Client schematic](/static/images/client-architecture.png)
 
-- Generate a JWT secret under the path `<datadir>/geth/jwtsecret`. This secret is needed
-  both by Geth and the CL client.
+For this two-client structure to work, consensus clients must be able to pass bundles of transactions to Geth
+to be executed. Executing the transactions locally is how the client validates that the transactions do not
+violate any Ethereum rules and that the proposed update to Ethereum's state is correct. Likewise, when the node
+is selected to be a block producer the consensus client must be able to request bundles of transactions from Geth to
+include in the new block. This inter-client communication is handled by a local RPC connection using the 
+[`engine` API][engine-api-link] which is part of the JSON-RPC API exposed by Geth. 
 
-- Open HTTP and WS endpoints on the authenticated port 8551.
+## Transition
 
-This is what it looks like by default:
-
-```shell
-geth --ropsten --datadir ~/.ropsten --override.terminaltotaldifficulty 50000000000000000
-```
-
-```terminal
-INFO [05-25|11:04:41.179] Starting Geth on Ropsten testnet...
-...
-WARN [05-25|11:04:41.520] Catalyst mode enabled                    protocol=eth
-INFO [05-25|11:04:41.627] Generated JWT secret                     path=/home/user/.ropsten/geth/jwtsecret
-INFO [05-25|11:04:41.628] WebSocket enabled                        url=ws://127.0.0.1:8551
-INFO [05-25|11:04:41.628] HTTP server started                      endpoint=127.0.0.1:8551 auth=true prefix= cors=localhost vhosts=localhost
-```
-
-### Engine API Authentication
-
-The listening address of the engine API is configurable. It is also possible to self-generate the JWT secret and feed the resulting file to Geth. To generate the secret, run:
-
-```shell
-openssl rand -hex 32 | tr -d "\n" > "/tmp/jwtsecret"
-```
-
-Now configure authentication using Geth flags:
-
-```shell
-geth --ropsten --datadir ~/.ropsten --authrpc.addr localhost --authrpc.port 8551 --authrpc.vhosts localhost --authrpc.jwtsecret /tmp/jwtsecret --override.terminaltotaldifficulty 50000000000000000
-```
-
-[engineapi]: https://github.com/ethereum/execution-apis/blob/main/src/engine/specification.md
+The transition from PoW to PoS will happen when a pre-announced total difficulty is reached by the chain. 
+This is different to other forks which usually occur at a certain scheduled block number. The total difficulty threshold 
+that will trigger the Merge is also known as the [*Terminal
+Total Difficulty* (TTD)](https://ethereum.org/en/glossary/#terminal-total-difficulty). In
+case of an emergency delay, the TTD can be overriden in Geth using the `--override.terminaltotaldifficulty` command-line
+flag. Once the merge block exists, Geth will no longer be able to follow the hed of the chain without a consensus
+client. If Geth is not connected to a consensus client it will simply stall at the merge block awaiting instructions
```

Comment:
This sentence is a little confusing because it starts by saying "if geth is not connected to CL" and ends "then it will wait for CL".